package.json

```diff
@@ -7,9 +7,9 @@
   "packageManager": "yarn@1.22.19",
   "dependencies": {
     "@popperjs/core": "^2.11.6",
-    "@rails/actioncable": "^6.0.0",
-    "@rails/activestorage": "^6.0.0",
-    "@rails/ujs": "^6.0.0",
+    "@rails/actioncable": "7.1.3",
```

Comment:
Ha van npm package megfelelője, akkor érdemes még a patch verziónak is megegyeznia. A mostani gem verzió a `7.1.3.4`, szuper lenne ha itt is `7.1.3-4` szerepelne.